### PR TITLE
ui2: Wallet List Scene 2.0 design

### PR DESCRIPTION
**Pull request by amanalili**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android

Screen Shot 2020-12-30 at 6 29 17 PM

Screen Shot 2020-12-30 at 10 29 29 PM

### CODE REVIEW

File: src/components/scenes/WalletListScene.js [hunk 1]
```diff
@@ -101,21 +101,23 @@ class WalletListComponent extends React.PureComponent<Props, State> {
...
+                <Fontello name="sort" size={theme.rem(1.5)} color={theme.iconTappable} />
               </TouchableOpacity>
             </View>
+          )}
+          {sorting && (
             <TouchableOpacity key="doneButton" style={styles.headerButtonsContainer} onPress={this.disableSorting}>
               <EdgeText>{s.strings.string_done_cap}</EdgeText>
             </TouchableOpacity>
-          </CrossFade>
+          )}
```
Comment (paullinator) [inline src/components/scenes/WalletListScene.js:120]: Why was cross fade removed?

File: src/components/themed/WalletListFooter.js [hunk 2]
```diff
@@ -45,20 +44,8 @@ class WalletListFooterComponent extends React.PureComponent<StateProps & ThemePr
...
-        </View>
-        <ButtonBox margin={0.5} onPress={Actions[Constants.PLUGIN_BUY]}>
-          <View style={styles.buyCryptoContainer}>
-            <View style={styles.buyCryptoImagesContainer}>
-              <Image style={styles.buyCryptoImages} source={{ uri: Constants.CURRENCY_SYMBOL_IMAGES.BTC }} resizeMode="cover" />
-              <Image style={styles.buyCryptoImages} source={{ uri: Constants.CURRENCY_SYMBOL_IMAGES.ETH }} resizeMode="cover" />
-              <Image style={styles.buyCryptoImages} source={{ uri: Constants.CURRENCY_SYMBOL_IMAGES.BCH }} resizeMode="cover" />
-            </View>
-            <EdgeText style={styles.buyCryptoText}>{s.strings.title_plugin_buy_sell}</EdgeText>
-          </View>
```
Comment (paullinator) [inline src/components/themed/WalletListFooter.js:60]: Remove string if unused

**Review by paullinator — APPROVED** [review]
Please address comments in another PR

File: src/components/scenes/WalletListScene.js (hunk 1 above)
Comment (amanalili) [reply src/components/scenes/WalletListScene.js:120]: Messes with the design because of it's restrictive style

**Pull request merged**